addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -100,6 +100,27 @@ def _get_mo_summary(self, production, components, operations):
         }
 
     def _format_state(self, record):
+        if record._name == 'mrp.production' and record.state in ('draft', 'confirmed') and record.move_raw_ids:
+            total_product_qty = defaultdict(float)
+            for move_raw_id in record.move_raw_ids.filtered(lambda l: l.product_id.detailed_type == 'product'):
+                total_product_qty[move_raw_id.product_id] += move_raw_id.product_uom_qty
+            producible_qty = record.product_qty
+            for move in record.move_raw_ids:
+                if move.product_id.detailed_type != 'product':
+                    continue
+                comp_producible_qty = float_round(
+                    ((move.reserved_availability + move.product_id.uom_id._compute_quantity(move.product_id.free_qty, move.product_uom)) / (total_product_qty[move.product_id]/record.product_uom_qty)),
+                    precision_digits=0, rounding_method='DOWN'
+                )
+                if comp_producible_qty == 0:
+                    producible_qty = 0
+                    break
+                producible_qty = min(comp_producible_qty, producible_qty)
```

Comment:
This is so wrong I am not going to waste any more time telling you how your code is wrong except that your UoM calculations are STILL wrong.

I will finish this task by myself.